test/e2e/README.md

```diff
@@ -0,0 +1,19 @@
+# E2E Testing
+
+This suite tests the end-to-end operation of the open-feature-operator.
+
+Tests are written with [kuttl](https://kuttl.dev/) and assertions are executed from a curl enabled Job.
+Ngnix reverse proxy is used as the workload where flagd get injected using OFO annotations.
+
+## Running and validating locally
+
+It is recommended to run and validate e2e test locally before opening a pull request.
+
+To run locally (commands are executed from the project root level),
+
+1. Build the operator locally - `docker build . -t open-feature-operator-local:validate`
```

Comment:
should we also provide a target in the Makefile for executing these commands? If yes, this can also be done in a follow up PR

Reply:
Good suggestion. I added this with the latest commit - https://github.com/open-feature/open-feature-operator/pull/521/commits/c6dbda979c69e9d8931efc7696f513346652bef0#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R77-R84